src/codeu/chat/common/User.java

```diff
@@ -52,11 +53,19 @@ public User read(InputStream in) throws IOException {
   public final String name;
   public final Time creation;
 
-  public User(Uuid id, String name, Time creation) {
+  /*
+   * The sentiment score is public since the only way to update it is by connecting to the natural
+   * language API, which only the server can do. As such, the client must request a new sentiment
+   * score from the server
+   */
+  public SentimentScore sentimentScore;
+
+  public User(Uuid id, String name, Time creation, SentimentScore score) {
 
     this.id = id;
     this.name = name;
     this.creation = creation;
+    this.sentimentScore = score;
```

Comment:
When a client is given the users, they will now be given each users sentiment score, as well. This is shown in the serializer of the user. As such, this parameter is really for the client to reconstruct the users.